modules/deploy/pages/console/linux/index.adoc

```diff
@@ -0,0 +1,7 @@
+= Deploy Redpanda Console on Linux
+:description: Platform-specific deployment instructions for Redpanda Console on Linux.
```

Comment:
minor, but it'd be nice if this was parallel structure with the other Deploy descriptions, so the index pages for Deploy/Redpanda and Deploy/Console look similar.